src/base/_mixins.scss

```diff
@@ -0,0 +1,36 @@
+@mixin size($width, $height) {
+  width: $width;
+  height: $height;
+}
+
+@mixin padding($top, $bottom, $left, $right) {
```

Comment:
can we mark all these as optional somehow? And check against null, similarly to the flexContainer mixin